subprojects/gradle/gradle-logger/src/main/kotlin/com/avito/logger/GradleLoggerFactory.kt

```diff
@@ -127,9 +132,25 @@ class GradleLoggerFactory(
             elasticConfig = ElasticConfigFactory.config(project),
             projectPath = project.path,
             pluginName = pluginName,
-            taskName = taskName
+            taskName = taskName,
+            verboseMode = readVerboseMode(project)
         )
 
+        @Suppress("UnstableApiUsage")
+        private fun readVerboseMode(project: Project): LogLevel {
+            return project.providers
+                .gradleProperty("avito.logging.verbose")
+                .forUseAtConfigurationTime()
+                .map {
+                    try {
+                        LogLevel.valueOf(it.toUpperCase(Locale.getDefault()))
+                    } catch (e: Throwable) {
```

Comment:
Let's fail in case of invalid value. 
It's hard to notice not working logging property.

Reply:
I am not sure if throwing error inside map is ok, but i did it